smithy-openapi/src/main/java/software/amazon/smithy/openapi/fromsmithy/OpenApiConverter.java

```diff
@@ -637,7 +637,7 @@ private OperationObject addOperationTags(
         for (SecuritySchemeConverter<? extends Trait> converter : context.getSecuritySchemeConverters()) {
             if (defaultAuthTraits.contains(converter.getAuthSchemeType())) {
                 List<String> result = createSecurityRequirements(context, converter, context.getService());
-                String authSchemeName = converter.getAuthSchemeId().toString();
+                String authSchemeName = converter.getAuthSchemeId().toString().replace("#", ".");
```

Comment:
This is repeated in lots of places -- let's move this to a single method to make it easier to not get wrong or forget about later.

Reply:
added getOpenApiAuthSchemeName() method.